src/aiida_quantumespresso/workflows/ph/base.py

```diff
@@ -34,9 +41,20 @@ def define(cls, spec):
         super().define(spec)
         spec.expose_inputs(PhCalculation, namespace='ph')
```

Comment:
We should also exclude the `qpoints` input of the wrapped `PhCalculation` here:

```suggestion
        spec.expose_inputs(PhCalculation, namespace='ph', exclude=('qpoints', ))
```

Reply:
One thought, not to be implemented here but to consider: What is more user friendly:

1. Having both the `qpoints` and `qpoints_distance` at the `PhBaseWorkChain` level, as is the case now?
2. Only adding an (optional) `qpoints_distance` input at the `PhBaseWorkChain` level, and leaving the `qpoints` input at the `PhCalculation` level?

Reply:
This i am not sure :D , i didnt want to change the logic thinking of how it could affect the workflows of aiida_quantumespresso_ph , @bastonero do you have an opinion on this ?

Reply:
Well honestly since the `PwBaseWorkChain` and other workchains has `qpoints` at the same level of `qpoints_distance`, I would put `qpoints` higher up. We should push the idea of using only `WorkChains` and not `CalcJobs`. But above of all, documentation is lacking ^^

Reply:
True true, my thinking was that we still specify other direct inputs at the calcjob level, so maybe it's more reasonable keeping it there, instead of pushing it higher. But I have no strong opinion.

Except the documentation part. That I fully agree 😓

Reply:
Yeah I understand the point. I just don't know what's the amount of people we annoy when changing patterns. My strong opinion is to have the same "patterns" over the different WorkChains, so one expects `qpoints` or `kpoints` at a certain level. Which one I don't know. But probably i would stick to moving it higher just for historical reasons, and then maybe having an other related issue where to discuss this, and put it e.g. in `5.0`?

Reply:
Fully agree, I wouldn't have suggested implementing it in this PR. Will give it some more thought and potentially open an issue. :)